TAC/Meetings/2022/2022-04-07/CCC_TAC_Minutes-2022-04-07.md

```diff
@@ -41,89 +42,117 @@
 
-1. **[Brian Warner]** Verify that third-party dependencies appear in the [CCC dependencies spreadsheet](https://docs.google.com/spreadsheets/d/1UKnbbGWXYLjnPZsox3zmYo59nv3XSXjePfas5E2fER0/edit#gid=0). (done)
 
-### Project overviews
+### Completed Action Items
+1. **[Dan Middleton]** Work with CCC Attestation to schedule annual review, tentatively for March 24, 2022.
 
-* Veraison
+1. **[Dave Thaler]** Open an issue requesting someone sync project benefits slides to the repo.
 
+1. **[Brian Warner]** Update the Google Sheet containing the [list of CCC dependencies](https://docs.google.com/spreadsheets/d/1UKnbbGWXYLjnPZsox3zmYo59nv3XSXjePfas5E2fER0/edit#gid=0) with the changed dependencies noted in the project review document.
 
-### TAC Tech Talk - Confidential Computing from the perspective of the network
+1. **[Brian Warner]** Add Enarx annual review to the agenda for March 10th TAC meeting.
 
+1. **[Brian Warner]** Add slide to this deck reminding the TAC to review PRs and issues, time permitting.
 
-### CCC-Attestation annual review
+1. **[Brian Warner]** Schedule a deep-dive into LFX code scanning for April 19, 2022.
+    * Confirmed, Shubhra Kar and David Deal from LF to present.
 
+### Project overviews
 
-### Common terminology
 
+### Interns
+* Nick Vidal (NV) update on Interns, slack #general discussion, help and guidance 
+* Deferr to next meeting when Dan Middleton will be participating 
 
 ### Outreach update
 
+* Ben Fischer (BF) proposed May webinar panel talk for The Future of Attestation
+	* Asking for moderator and participants
+	* (DT) mentioned the list of past TAC talks as possible outreach webinar topics, Mark N, Simon J, Dave T, Henk, Thomas F
+	* (BF) to organize
+* (DT) discussed needed updates to the Trusted Computing wikipedia page, also create a CCC or Confidential Computing page (none currently exists)
+	* (NV) suggested the interns do an initial wikipedia page for CC, (NV) will work with them 
+	* (NV) authoring a blog, will announce soon	 
 
-### Future topics
+### Common terminology
+ * (DT) discussed Google Doc for terminology
+ * Eric Voit (EV) raised multi node question as previously discussed with regard to confidential cloud, multi node is a hard problem to define, will defer to a future version
+ * Steve Van Lare (SVL) will respond to comments in document
+ * (DT) proposed and (SVL) discussed confidential workload/application - protects all legs of CC, (at rest, in motion, and in use) 
+ * (EV) github process closure (DT) proposed PDF to whitepaper, Draft then to LF creative team for editing process, goal to be firmed up by next meeting, draft by second meeting will be signed off, I'll find out creative services timeline
```

Comment:
```suggestion
 * (EV) github process closure (DT) proposed PDF to whitepaper, Draft then to LF creative team for editing process, goal to be firmed up by next meeting, draft by second meeting will be signed off, KT to find out creative services timeline
```
Also this means KT has an action item that should be listed in the action item list